Source/Models/FORMGroup.m

```diff
@@ -54,6 +56,10 @@ - (instancetype)initWithDictionary:(NSDictionary *)dictionary
     return self;
 }
 
+- (void)updateGroupWithCollapsed:(BOOL)collapsed {
```

Comment:
@jeffleeismyhero Looks like you could remove this method declaration, I can't find any references to it.